sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -66,6 +66,7 @@ class EventHubConsumerClient(ClientBaseAsync):
      :class:`EventHubSharedKeyCredential<azure.eventhub.aio.EventHubSharedKeyCredential>`, or credential objects
      generated by the azure-identity library and objects that implement the `get_token(self, *scopes)` method.
     :type credential: ~azure.core.credentials_async.AsyncTokenCredential or ~azure.core.credentials.AzureSasCredential
+     or ~azure.core.credentials.AzureNamedKeyCredential
```

Comment:
AzuerNamedKeyCredential is in azure-core and doesn't have an async implementation (since it's unnecessary)

for token credentials, we have an async implemtnetation because we needed an  "async get_token" method in this case